app/javascript/controllers/iiif_auth_restriction_controller.js

```diff
@@ -0,0 +1,45 @@
+import { Controller } from "@hotwired/stimulus"
+
+// Display auth controls for the file_auth_controller.js
+export default class extends Controller {
+  static targets = ["locationRestriction", "locationRestrictionMessage", "messagePanel", "loginPanel",
+                    "loginButton", "loginMessage"]
+
+  // Bound to auth-denied CustomEvents
+  displayMessage(event) {
+    const detail = event.detail
+    this.locationRestrictionTarget.hidden = false
+    this.locationRestrictionMessageTarget.innerHTML = this.#retrieveAuthResponseMessage(detail.authResponse)
```

Comment:
Not sure if this is commented out because of the #

Reply:
that's not a comment. That's the sigil for a private method in javascript https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties

Reply:
Cool!